server/verify.go

```diff
@@ -59,7 +59,7 @@ type gceInstanceInfo struct {
 	ProjectID          string `asn1:"utf8"`
 	InstanceID         int64
 	InstanceName       string                `asn1:"utf8"`
-	SecurityProperties gceSecurityProperties `asn1:"optional"`
+	SecurityProperties gceSecurityProperties `asn1:"explicit"`
```

Comment:
Should this be marked explicit and optional?

Reply:
added